<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProfileDialogBase</class>
 <widget class="QDialog" name="ModifyProfileDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Profile Name</string>
     </property>
     <property name="buddy">
      <cstring>leProfile</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Exchange Server</string>
     </property>
     <property name="buddy">
      <cstring>leServer</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Domain</string>
     </property>
     <property name="buddy">
      <cstring>leDomain</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leDomain">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>Active Directory domain</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Username</string>
     </property>
     <property name="buddy">
      <cstring>leUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="leUsername">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>Active Directory username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Password</string>
     </property>
     <property name="buddy">
      <cstring>lePassword</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="leProfile"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="leServer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>Hostname of Exchange server, e.g. mail.biz.com</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lePassword">
     <property name="whatsThis">
      <string>Enter password to fetch details</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lePassword</tabstop>
  <tabstop>leServer</tabstop>
  <tabstop>leDomain</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifyProfileDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifyProfileDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
